src/db_stats.cpp

```diff
@@ -131,12 +134,12 @@ class StatsMetrics
         pending_time_.metadata.scope(metricq::Metadata::Scope::last);
         pending_time_.metadata.rate(rate);
 
-        active_time_.metadata.unit("s");
-        active_time_.metadata.quantity("time");
-        active_time_.metadata.description(
-            fmt::format("average time {}-requests were being processed", read_or_write));
-        active_time_.metadata.scope(metricq::Metadata::Scope::last);
-        active_time_.metadata.rate(rate);
+        active_utilization_.metadata.unit("");
+        active_utilization_.metadata.quantity("utilization");
+        active_utilization_.metadata.description(
+            fmt::format("average utilization for {}-requests were being processed", read_or_write));
```

Comment:
I'll try to sharpen the definition, but I'm not perfectly happy.

```suggestion
            fmt::format("fraction of time spent on processing {}-requests", read_or_write));
```